### DISCUSSION TITLE
How to prevent the agent from displaying the details of tool invocation when invoking the tool?

### DISCUSSION THREAD

**Discussion by huziget** [langgenius/dify#12151] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

When I am using the agent and then invoking my tool, the information of the tool invocation is displayed. I only want a concise reply. How can I configure this?
<img width="613" alt="6811735281974_ pic" src="https://github.com/user-attachments/assets/dcef77c6-2b0f-4572-8da6-fe63f68226f9" />

**Comment by pigracing** [comment]
在提示词那里去要求不要输出json格式的内容，但还是偶尔会输出

**Reply by zhengyuhe123** [reply]
我调用的工具的key是放在变量里面的，不想把这个调用的详细的信息返回给用户。找了一圈也没有设置的地方....

**Comment by a1113** [comment]
It's definitely possible to add a detailed description to the model cue, which will undoubtedly enhance your understanding and make it easier to try it out. You can provide specific details about the model's functionality, any prerequisites for using it, and potential outcomes or benefits. This additional information can be incredibly helpful for those who are new to the concept or need a clearer picture of what to expect. So, go ahead and give it a try with the added description to fully explore its capabilities!
